nostr/relay.py

```diff
@@ -75,12 +79,13 @@ def _on_close(self, class_obj, status_code, message):
     def _on_message(self, class_obj, message: str):
         if self._is_valid_message(message):
             self.message_pool.add_message(message, self.url)
-    
+
     def _on_error(self, class_obj, error):
         pass
 
     def _is_valid_message(self, message: str) -> bool:
-        if not message or message[0] != '[' or message[-1] != ']':
+        message = message.strip("\n")
```

Comment:
This is the only change in `nostr/*.py` that differs from origin, all other changes are formatting.